blog/_posts/2018-04-30-in-a-nutshell.md

```diff
@@ -182,6 +182,13 @@ that so far required advanced macro code to define. In particular:
 
  - There will be a way to do typeclass derivation a la [Kittens](https://github.com/milessabin/kittens), [Magnolia](https://github.com/propensive/magnolia), or [scalaz-deriving](https://gitlab.com/fommil/scalaz-deriving) that does not need macros. We are currently evaluating the alternatives. The primary goal is to develop a scheme that is easy to use and that performs well at both compile- and run-time. A second goal is generality, as long as it does not conflict with the primary goal.
 
+## Meta Programming in the Large
```

Comment:
Thanks for adding this section! It is good to stress these are complementary technologies.